### ISSUE TITLE
Ship automatic schema update

### ISSUE THREAD

**Issue by bebehei** [jjethwa/icinga2#118]
Hey @jjethwa 

I hadn't had the time to update to 2.8 yet. But https://www.icinga.com/2017/11/17/icinga-2-v2-8-0-released/ mentions the DB IDO schemas got updated. This renders the complete DB unusable and requires manual intervention.

What about making this easier by shipping an additional command like `docker exec -ti <container> icinga2-ido-update`?

**Comment by jjethwa** [comment]
Great idea @bebehei  😄

**Comment by ruudboon** [comment]
Any updates on this? Having some issues now with the Director 1.4.3.

**Comment by bebehei** [comment]
@ruudboon No, no progress. Sorry. You have to apply the schema update file manually.

**Comment by ruudboon** [comment]
@bebehei ok, no problem! Thank you for creating this project!
